components/pages/Messages/Messages.js

```diff
@@ -174,25 +174,212 @@ export function Messages({ fromUserId }) {
+                                  </div>
+                                </div>
+                              )}
+
+                              <button
+                                className={cx(styles.threeDotsButton, {
+                                  [styles.isClicked]:
+                                    messageIndex === index && isDotsClicked,
+                                })}
+                                onClick={() => {
+                                  setMessageIndex(
+                                    messageIndex === index ? undefined : index
+                                  );
+                                  if (messageIndex === index) {
+                                    setIsDotsClicked(true);
+                                  }
+                                }}
+                              >
+                                <svg
+                                  className={cx(styles.threeDotsSVG, {
+                                    [styles.isClicked]:
+                                      messageIndex === index && isDotsClicked,
+                                  })}
+                                  aria-label="Comment options"
+                                  fill="#8e8e8e"
+                                  height="24"
+                                  role="img"
+                                  viewBox="0 0 24 24"
+                                  width="24"
+                                >
+                                  <circle cx="12" cy="12" r="1.5"></circle>
+                                  <circle cx="6.5" cy="12" r="1.5"></circle>
+                                  <circle cx="17.5" cy="12" r="1.5"></circle>
+                                </svg>
+                              </button>
+                            </div>
+                          </div>
+                        )}
+                      </div>
+                      {message.is_liked_by_user && (
+                        <div
+                          className={cx(styles.likeIconContainer, {
+                            [styles.isSent]: message.direction === 'sent',
+                          })}
+                        >
+                          <svg
+                            aria-label="like-icon"
+                            height="14"
+                            role="img"
+                            viewBox="0 0 48 48"
+                            width="14"
+                          >
+                            <path
+                              className={styles.likeIcon}
+                              d="M34.6 3.1c-4.5 0-7.9 1.8-10.6 5.6-2.7-3.7-6.1-5.5-10.6-5.5C6 3.1 0 9.6 0 17.6c0 7.3 5.4 12 10.6 16.5.6.5 1.3 1.1 1.9 1.7l2.3 2c4.4 3.9 6.6 5.9 7.6 6.5.5.3 1.1.5 1.6.5s1.1-.2 1.6-.5c1-.6 2.8-2.2 7.8-6.8l2-1.8c.7-.6 1.3-1.2 2-1.7C42.7 29.6 48 25 48 17.6c0-8-6-14.5-13.4-14.5z"
+                            ></path>
+                          </svg>
```

Comment:
This should be just a heart emoji. Like this ❤️. Check how the like looks on my machine.

![image](https://user-images.githubusercontent.com/17054134/148146461-6984522f-7ff7-4c3f-bc52-4cf796f84184.png)
```suggestion
                          ❤️
```

Reply:
On it.